SubmittingPatches.rst

```diff
@@ -133,7 +133,7 @@ Some negative examples (how *not* to title a commit message)::
      bug fix for driver X
      fix issue 99999
 
-Further to the last negative example ("fix issue 99999"), see `Fixes line`_.
+Further to the last negative example ("fix issue 99999"), see `Fixes line(s)`_.
```

Comment:
i think Nathan intended to put "fixes line" instead. as he added an anchor right before the section of "Fixes line(s)".